Source/WebCore/rendering/style/TextSizeAdjustment.h

```diff
@@ -64,7 +64,7 @@ class AutosizeStatus {
 
     constexpr bool contains(Fields) const;
 
-    constexpr bool operator==(const AutosizeStatus& other) const { return fields() == other.fields(); }
+    friend constexpr bool operator==(const AutosizeStatus&, const AutosizeStatus&) = default;
```

Comment:
Since this is just an `OptionSet`, clearly it can just be `AutosizeStatus` without the `const&`, but I am unclear on the best rule we should follow about when to do that.